Std/Tactic/SolveByElim.lean

```diff
@@ -80,7 +78,7 @@ calls to `apply` succeeded or failed.
 def applyTactics (cfg : ApplyConfig := {}) (transparency : TransparencyMode := .default)
     (lemmas : List Expr) (g : MVarId) : Nondet MetaM (List MVarId) :=
   (Nondet.ofList lemmas).filterMapM fun e => observing? do
-    withTraceNode `Meta.Tactic.solveByElim (return m!"{emoji ·} trying to apply: {e}") do
+    withTraceNode `Meta.Tactic.solveByElim (return m!"{Except.emoji ·} trying to apply: {e}") do
```

Comment:
It seems to me that dot notation would be preferable for these `Except.emoji ·` expressions (i.e., `·.emoji`).